### PR TITLE
Introduces more configuration for rails i18n

### PR DESCRIPTION
**Pull request by kbacha**
* Documents ignoring exceptions
* Allows the Regexp to be specified by config
* Can choose between 3 different sentence structures to evaluate

### CODE REVIEW

File: lib/rubocop/cop/i18n/rails_i18n/decorate_string.rb [hunk 1]
```diff
@@ -21,8 +21,59 @@ module RailsI18n
         #   t("result_is_good")
         #   I18n.t("result_is_good")
         #
+        # There are several options for configuration.
+        #
+        # @example IgnoreExceptions: true
+        #   # OK
+        #
+        #   raise "Some string sentence"
```
Comment (ghiculescu) [inline lib/rubocop/cop/i18n/rails_i18n/decorate_string.rb]: is this behaviour implemented? i'm not seeing it.

File: lib/rubocop/cop/i18n/rails_i18n/decorate_string.rb [hunk 2]
```diff
@@ -21,8 +21,59 @@ module RailsI18n
...
+        #
+        # @example Regexp: ^only-this-text$
+        #
+        #   # bad
+        #
+        #   "only-this-text"
+        #
+        #   # good
+        #
+        #   "Any other string is find now"
```
Comment (ghiculescu) [inline lib/rubocop/cop/i18n/rails_i18n/decorate_string.rb]:
```suggestion
        #   "Any other string is fine now"
```

**Comment by kbacha** [comment]
Sorry, I would have gotten to this but I’ve been on family leave with a new born :)

**Comment by highb** [comment]
@kbacha Oh no worries at all! Congratulations and enjoy your time off.

**Comment by tphoney** [comment]
thanks for the great work !!

**Pull request merged**